### PR TITLE
Players can now be activated or deactivated

### LINKED ISSUE #14: Add the ability to mark a user active or inactive
Instead of having to remove a player to make them not contribute to the player count, it would be nice to be able to toggle them active/inactive.

### PR DESCRIPTION
**Pull request by Serneum**
Closes #14 

@daveshah @mbramson @ericworkman @joelbyler please tear this apart

### CODE REVIEW

File: web/controllers/page_controller.ex [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
   use Nermesterts.Web, :controller
   alias Nermesterts.Game
+  alias Nermesterts.Phrase
   alias Nermesterts.Player
 
   def index(conn, _params) do
     games = Nermesterts.Repo.all(Game)
-    players = Nermesterts.Repo.all(Player)
-    num_players = length(players)
+    active_players = Nermesterts.Repo.all(from p in Player, where: p.active == true)
```
Comment (Serneum) [inline web/controllers/page_controller.ex]: Ideally, I'd like to keep all of the players together and just filter them on the page

File: web/templates/player/index.html.eex [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+    </tr>
+  </thead>
+  <tbody>
+<%= for player <- @active_players do %>
+    <tr>
+      <td><%= player.name %></td>
+
+      <td class="text-right">
+        <%= link "Edit", to: player_path(@conn, :edit, player), class: "btn btn-default btn-xs" %>
+        <%= link "Deactivate", to: player_path(@conn, :update, player, %{player: %{active: !player.active}}), method: :put, class: "btn btn-default btn-xs" %>
```
Comment (Serneum) [inline web/templates/player/index.html.eex]: This was a pain. If you know of a better way to call `update` with the updated player params, please let me know

File: web/models/player.ex [hunk 3]
```diff
@@ -12,7 +13,7 @@ defmodule Nermesterts.Player do
   """
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:name])
+    |> cast(params, [:name, :active])
     |> validate_required([:name])
```
Comment (Serneum) [inline web/models/player.ex]: Sounds good to me. This was just what Phoenix generated

File: web/templates/player/index.html.eex (hunk 2 above)
Comment (lojack) [inline web/templates/player/index.html.eex]: Should there be a separate route just for activating/deactivating players? Only reason I suggest this is the permissions for activating/deactivating should be different than a player updating their own profile.

Comment (Serneum) [inline web/templates/player/index.html.eex]: Activating/Deactivating would just be for the current day and would be set by that player. Does that change your opinion on the routes?

Comment (Serneum) [inline web/templates/player/index.html.eex]: @daveshah you had mentioned how to properly do this ages ago. Can you explain again?

**Pull request merged**